ocs_ci/framework/conf/default_config.yaml

```diff
@@ -114,3 +114,8 @@ ENV_DATA:
 # This section is related to upgrade
 UPGRADE:
   upgrade_to_latest: true
+  # This default image for ocp upgrade based on GA build
+  ocp_upgrade_version: "4.3.3"
```

Comment:
This should be just a commented value mentioned as an example. We don't want to have this hardcoded as by default you should use the dynamic approach.
`# ocp_upgrade_version: "4.3.3"`